src/main/java/net/neoforged/neoforge/client/loading/NoVizFallback.java

```diff
@@ -1,44 +0,0 @@
-/*
- * Copyright (c) Forge Development LLC and contributors
```

Comment:
Remember to remove the workarounds in:
- the check split sources task 
- the package info generation task